warehouse/schema.go

```diff
@@ -19,7 +20,7 @@ type bundleEvent struct {
 	EventModError          int64
 	EventModSuspicious     int64
 	IndvId                 int64
-	PageClusterId	       int64
+	PageClusterId	         int64
```

Comment:
nit: weird whitespace

Reply:
This was actually weird in my IDE prior to this. I think we have mixed tabs and spaces. And my tab width is different than ya'lls. Fixing.

Reply:
Ran `go fmt` on the file. We should probably setup some presubmit hooks etc... on these repos.